### ISSUE TITLE
Copy only changed files

### ISSUE THREAD

**Issue by slavafomin** [ryanmcgrath/wrench-js#110]
Hello!

I have a project with extremely big directories. It would be a tremendous improvement if **wrench** would be able to compare files from source and target directory using timestamp and/or filesize.

Thanks!

**Comment by igorshubovych** [comment]
Being honest `wrench` was not updated for 2 years, because the project is looking for maintainer.

If I were you, I would implement this routine myself or find another solution.

**Comment by ryanmcgrath** [comment]
^this pretty much. Was built as a stop gap for very early versions if Node, better libraries exist now. I only keep it available because for some reason an unholy amount of projects still list it as a dependency.






On Fri, Mar 11, 2016 at 2:00 PM -0800, "Igor Shubovych" <notifications@github.com> wrote:












Being honest wrench was not updated for 2 years, because the project is looking for maintainer.



If I were you, I would implement this routine myself or find another solution.

**Comment by slavafomin** [comment]
I see, @ryanmcgrath , you should add a deprecation notice to the README and use https://docs.npmjs.com/cli/deprecate to notify the users during module installation then, I think. Maybe even add a link to the most suitable alternative.

**Comment by ryanmcgrath** [comment]
Pull request it and I will? I'll likely forget due to a ridiculously packed schedule these days, but happy to push it up if so. Alternatively, if anyone wants to take over ownership of the library I'm happy to help facilitate this. I put out a call on Twitter some time ago seeing if anyone was interested but sadly found no takers.






On Sat, Mar 12, 2016 at 2:29 AM -0800, "Slava Fomin" <notifications@github.com> wrote:












I see, @ryanmcgrath , you should add a deprecation notice to the README and use https://docs.npmjs.com/cli/deprecate to notify the users during module installation then, I think. Maybe even add a link to the most suitable alternative.

**Comment by slavafomin** [comment]
Yeah, I think all professional developers are pretty busy these days. A lot of work needs to be done )
--
Sent from myMail app for Android Saturday, 12 March 2016, 03:31PM +03:00 from Ryan McGrath < notifications@github.com> :

>Pull request it and I will? I'll likely forget due to a ridiculously packed schedule these days, but happy to push it up if so. Alternatively, if anyone wants to take over ownership of the library I'm happy to help facilitate this. I put out a call on Twitter some time ago seeing if anyone was interested but sadly found no takers.
>
>
>
>
>
>
